RecoEgamma/EgammaTools/plugins/EGExtraInfoModifierFromDB.cc

```diff
@@ -604,8 +604,16 @@ void EGExtraInfoModifierFromDB::modifyObject(reco::Photon& pho) const {
   if (iseb) {
     EBDetId ebseedid(theseed->seed());
     eval[26] = pho.e5x5()/theseed->energy();
-    eval[27] = ebseedid.ieta();
-    eval[28] = ebseedid.iphi();
+    int ieta = ebseedid.ieta();
+    int iphi = ebseedid.iphi();
+    eval[27] = ieta;
+    eval[28] = iphi;
+    eval[29] = (ieta-1*abs(ieta)/ieta)%5;
```

Comment:
```((int) ieta - std::copysign(1, ieta) ) )%5 ```
or ``` ( ieta - (ieta > 0 ? 1 : -1)%5 ``` 
this will not generate warnings that there may be a division by zero
same below